### PR TITLE
feat(getinfo): Add class instance specifically to remove header

### PR DESCRIPTION
**Pull request by mikkelkp**

### CODE REVIEW

**Comment by mostaphaRoudsari** [comment]
Hi @mikkelkp! I'm not clear how we may want to use this in honeybee-radiance.

Have you seen the `parse_header` function?

https://github.com/ladybug-tools/honeybee-radiance/blob/1bf740e32ff6448f0bf6cbc7390e7fcb8151e4c7/honeybee_radiance/reader.py#L161

And also how we deal with a header right now:

https://github.com/ladybug-tools/honeybee-radiance/blob/54027500381018b218bafd9d67b0ad398b135327/honeybee_radiance/cli/util.py#L84

Is your implementation going to make things easier?

**Comment by mikkelkp** [comment]
Hi @mostaphaRoudsari. I was thinking that in cases where we want to remove the header we might as well pipe the Radiance command to "getinfo -" so we get all in one go.

Example:
dctimestep view transmission daylight sky | rmtxop -c 47.4 119.9 11.6 - | getinfo - > grid.ill

File: honeybee_radiance_command/getinfo.py [hunk 1]
```diff
@@ -30,13 +30,25 @@ class Getinfo(Command):
         * input
     """
 
-    __slots__ = ('_input')
+    __slots__ = ('_input', '_header')
 
-    def __init__(self, options=None, output=None, input=None):
+    def __init__(self, options=None, output=None, input=None, header=None):
```
Comment (mostaphaRoudsari) [inline honeybee_radiance_command/getinfo.py:35]: Should we set the default value to `False`? Would a keyword like `remove_header` or `keep_header` be more explicit?

File: honeybee_radiance_command/getinfo.py [hunk 2]
```diff
@@ -30,13 +30,25 @@ class Getinfo(Command):
...
         """Initialize Command."""
         Command.__init__(self, output=output)
         self._input = input
         self.options = options
+        self._header = header
+
+    @classmethod
+    def header(cls, options=None, output=None, input=None):
```
Comment (mostaphaRoudsari) [inline honeybee_radiance_command/getinfo.py:43]: Similarly I think we should call this `no_header` or something like that to be more explicit.

File: honeybee_radiance_command/getinfo.py (hunk 1 above)
Comment (mostaphaRoudsari) [reply honeybee_radiance_command/getinfo.py:35]: We should add an explanation to docstrings

**Review by mostaphaRoudsari — COMMENTED** [review]
Hi @mikkelkp, I have conflicting feelings about this but since this is a small change it is probably fine. I left some minor comments.

**Comment by mikkelkp** [comment]
Hi @mostaphaRoudsari. I removed the argument from `__init___` and instead set the attribute to `False`. I also added warnings when using 'a' and 'd'.

**Review by mostaphaRoudsari — APPROVED** [review]
Thanks @mikkelkp!

**Pull request merged**